packages/material-ui-system/src/Box/Box.spec.tsx

```diff
@@ -36,7 +36,7 @@ function GapTest() {
       display: 'flex',
       alignItems: 'center',
       flex: '1 0',
-      gridGap: (theme) => theme.spacing(2),
+      gap: '16px',
```

Comment:
Why this line need to change? I guest `.spec.tsx` is to test the type.

Reply:
The `Box` inside the `@material-ui/system` does not have an opinion on the theme structure, that is why it was changed.

Reply:
I would complement the answer with: the default theme in the system is currently empty, so it breaks.

I personally think that Box in the system should expect the same theme structure as in the core, so it can be used as a specification, used and learned once by the community, maybe even building third-party components based on it.

Reply:
> I personally think that Box in the system should expect the same theme structure (or a subset) as in the core, so it can be used as a specification, used and learned once by the community, maybe even building third-party components based on it.

Agree. On this note, should the system's theme expect maybe optional structure of the mui theme?

Reply:
By subset, I meant something like:

```jsx
{
  breakpoints: {
    values: {},
  },
  direction: 'ltr',
  mixins: {},
  components: {},
  palette: {
    mode: 'light',
  },
  shadows: {},
  typography: {},
  spacing: 8, or [] or () => number
  shape: {
    borderRadius:
  },
  transitions: {},
  zIndex: {},
}
```

Reply:
I will try as a follow-up to polish this. The goal would be to make the core Theme extend this one and make it stricter.